api/src/action/action.entity.ts

```diff
@@ -0,0 +1,73 @@
+
+import {
+    BeforeUpdate,
+    Column,
+    Entity,
+    OneToOne,
+    PrimaryGeneratedColumn,
+    JoinColumn,
+    ManyToMany,
+    Unique,
+    Index,
+    OneToMany
+} from 'typeorm';
+
+import { WorkflowStep } from '../wf-step/wf-step.entity';
+import { Workflow as WorkflowEntity } from "../workflow/workflow.entity"
+import { WfActionExecutionEntity } from "../wf-action-execution/wf-action-execution.entity";
+import { WfStepActionExecutionEntity } from '../wf-step-action-execution/wf-step-action-execution.entity';
+
+
+@Entity("action")
+export class ActionEntity {
```

Comment:
It is important to have columns as i have described here
https://workflowy.com/s/workflow-structure-v/uyoW3sHlRAgvNW9o

`alias` and `version` are needed to support versioning.
`id` should be a UUID as the amount of actions can be pretty big.
`body` should be of type `TEXT` - unlimited string